src/inmanta/data/__init__.py

```diff
@@ -3131,7 +3134,7 @@ async def log_instance_creation(
                 {cls.table_name()}
                 (id, tid, process, name, expired)
                 VALUES ($1, $2, $3, $4, null)
-                ON CONFLICT ON CONSTRAINT {cls.table_name()}_unique DO UPDATE
+                ON CONFLICT ON CONSTRAINT {cls.table_name(False)}_unique DO UPDATE
```

Comment:
Because the name of the constrain doesn't contain the schema. More than that, I think a constraint name can't contain a '.'. But I'm not sure of the later one.

Reply:
I see.